tests/test_runtime_gc.cpp

```
@@ -21,16 +20,16 @@ struct ArcHeader {
 
 // Allocate an ARC object with the given data size.
 void *gcAllocObj(size_t dataSize) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)